how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,135 @@
+
+But I tried… I tried to keep up with it for a few months and then I was like: "I don't get it! fuck this shit!" 
+
+"I have been doing design _and_ development 50/50 for all of my career, so now it's time to say goodbye to one of these and focus on the other." And since I got _so_ scared of The New Frontend World, I fled to the design department of [MyOnlineStore](https://www.myonlinestore.com/). This was in early 2018.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan-drawing.gif)
+
+_Yes, this is me doing design_
+
+Unfortunately, after a year of doing (UX) design, I found out this was the wrong choice. After years of intensive therapy and running straight into a black hole that was another potential burnout, I came to the conclusion that design was not for me. I am able to design, but I can't handle the stress that goes with it. I felt like I had to answer and fight for all the design choices I made. And since my UX designing is mostly driven by instinct, this was really hard for me. And on top of that it all felt very personal. Maybe I can handle it today or in the near future, with all the things I have learned since, but not then. It fueled my insecurity enormously.
+
+And then in late 2018, I decided to go back to frontend development. There I was, a frontend dinosaur in a modern world.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_dino.png)
+
+_Yes this is me._
+
+## How to get up to speed? 
+
+Now I faced a big challenge. I didn't want to go extinct like a real dinosaur and stick to my old habits, so how do you get up to speed after being paused for a few years in a world that changed (and is still changing) so rapidly? The first thing to overcome was my aversion for The New Frontend World™️. I tried to look at React and all other new tools with a fresh and unbiased view.
+
+While I was away in design world a lot has changed within our codebase, but also in The New Frontend World™️. The tools we used improved or were replaced with better alternatives. The tools and codebase became more solid and mature.
+
+Then there was news: [React Hooks](https://reactjs.org/docs/hooks-overview.html#%EF%B8%8F-effect-hook) was about to be released in React 16.8. This made code _so_ much more readable that I started thinking "Hmm, maybe I can do this".
+
+I don't want to go all geeky on the differences between hooks and classes and why hooks are so awesome, especially since there are [excellent](https://reactjs.org/docs/hooks-intro.html) articles [written](https://itnext.io/whats-the-big-deal-with-react-hooks-d73145eb14e0) on this [subject](https://medium.com/frontmen/react-hooks-why-and-how-e4d2a5f0347). If you like to learn more about this, you should check [them](https://reactjs.org/docs/hooks-overview.html) out.
```

Comment:
```suggestion
I don't want to go all geeky on the differences between hooks and classes and why hooks are so awesome, especially since there are many [excellent](https://reactjs.org/docs/hooks-intro.html) articles [written](https://itnext.io/whats-the-big-deal-with-react-hooks-d73145eb14e0) on this [subject](https://medium.com/frontmen/react-hooks-why-and-how-e4d2a5f0347). If you would like to learn more about this, you should check [them](https://reactjs.org/docs/hooks-overview.html) out.
```